### PR TITLE
[okta] Sync okta package with beats

### PR DESCRIPTION
**Pull request by marc-gr**
## What does this PR do?

Syncs okta package with changes made in beats

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/okta/_dev/deploy/docker/docker-compose.yml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-version: '2.3'
-services:
-  okta:
-    image: alpine
-    volumes:
-      - ./sample_logs:/sample_logs:ro
-      - ${SERVICE_LOGS_DIR}:/var/log
-    command: /bin/sh -c "cp /sample_logs/* /var/log/"
```
Comment (P1llus) [inline packages/okta/_dev/deploy/docker/docker-compose.yml:8]: Was this part meant to be removed?

File: packages/okta/data_stream/system/_dev/test/pipeline/test-system-events.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (P1llus) [inline packages/okta/data_stream/system/_dev/test/pipeline/test-system-events.json:1]: Most if not all golden files should usually start with the package name, similar to the old file that this was replaced with. Up to you if you want to change it though.

File: packages/okta/data_stream/system/agent/stream/httpjson.yml.hbs [hunk 3]
```diff
@@ -1,32 +1,38 @@
-{{#if api_key}}
-api_key: {{api_key}}
-{{/if}}
-authentication_scheme: {{authentication_scheme}}
```
Comment (P1llus) [inline packages/okta/data_stream/system/agent/stream/httpjson.yml.hbs:4]: Is authentication scheme not needed anymore?

**Review by P1llus — APPROVED** [review]
Small comments, again we can always pick up on them later if you feel they are not related, rest is LGTM :)

File: packages/okta/data_stream/system/agent/stream/httpjson.yml.hbs (hunk 3 above)
Comment (marc-gr) [reply packages/okta/data_stream/system/agent/stream/httpjson.yml.hbs:4]: was exposed only to expose the httpjson v1 config options, now is using the new one

**Pull request merged**